src/style.css

```diff
@@ -0,0 +1,135 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+body,
+h2,
+p {
+  margin: 0;
+  padding: 0;
+
+  --main-accent-color: #060b35;
+  --secondary-color: #616070;
+}
+
+.card {
+  box-sizing: border-box;
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  padding: 0 3px;
+  margin-top: 16px;
+  width: 160px;
+  height: 134px;
+}
+
+.card__title {
+  margin-top: 40px;
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: var(--main-accent-color);
+}
+
+.card__product-code {
+  margin-top: 4px;
+  font-family: "Roboto", sans-serif;
```

Comment:
No needs to use this rule, everywhere if you pass it to html/body
Check all over the page
```suggestion
```